tiledb/cloud/tasks.py

```diff
@@ -52,6 +52,12 @@ def tasks(
     """
     api_instance = client.client.tasks_api
 
+    if namespace is None:
+        if config.user is None:
+            config.user = client.user_profile()
```

Comment:
This is a kind of unexpected mutation of global state—I think a function to do this get-or-default probably belongs in config.py, so that it's more predictable what is going to happen there.